routes/addUser.js

```diff
@@ -0,0 +1,23 @@
+require('../models/user.js');
+var auth = require("../modules/auth");
+var mongoose = require("mongoose");
+var User = mongoose.model('User');
+
+module.exports = function(req, res) {
+    var infos = req.body;
+    var user;
+    user = new User({
+        cip: infos.cip,
+        prenom: infos.prenom,
+        nom: infos.nom,
+        email: infos.email,
+        concentration: infos.concentration,
+        promo: infos.promo
+    });
```

Comment:
For development's sake, please add a line to grant admin rights to the first registered user.
It will all save us trouble on each new deployment.
You may enclose it in a development override bool in config.